### PR TITLE
Adds action workflow

### LINKED ISSUE #1: Publish protrackor-junit-xml-plugin as npm package
To make it more convenient we want to publish this plugin as an npm package.

### PR DESCRIPTION
**Pull request by JeroenKnoops**
Closes #1

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1165 @@
+{
```
Comment (arsal123) [inline package-lock.json:1]: Do we really need to add package-lock.json file

**Review by arsal123 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (JeroenKnoops) [reply package-lock.json:1]: I think it's best practice to include the `package-lock.json` or `yarn.lock` file. Then everybody is sure what dependencies are pulled in.
There is some debate about it, but most projects seem to check them in.

See also https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Comment (arsal123) [reply package-lock.json:1]: ok I researched on it and found that it is recommended to have this file checked in

Comment (arsal123) [reply package-lock.json:1]: but probably not npm-shrinkwrap.json so this seems all good

**Pull request merged**